build.rs

```diff
@@ -59,6 +62,64 @@ fn main() {
     }
     cc.files(&files).compile("grumpkin_msm");
 
+    // ============================================================
+    // ====================== PASTA BINDINGS ======================
+    // ============================================================
```

Comment:
All the repeated code looks really weird, but I couldn't get anything to compile without doing it like this.